src/utils/RegHubUtil.js

```diff
@@ -117,59 +121,97 @@ module.exports = {
     repositoryServerActions.reposLoading({repos: []});
 
     hubUtil.request({
-      url: `${REGHUB2_ENDPOINT}/namespaces/`,
+      url: `${REGHUB2_ENDPOINT}/namespaces/`
     }, (error, response, body) => {
       if (error) {
         repositoryServerActions.error({error});
-        if (callback) { callback(error); }
-        return;
+        if (callback) {
+          return callback(error);
+        }
+        return null;
       }
 
       if (response.statusCode !== 200) {
         let generalError = new Error('Failed to fetch repos');
         repositoryServerActions.error({error: generalError});
-        if (callback) { callback({error: generalError}); }
-        return;
+        if (callback) {
+          return callback({error: generalError});
+        }
+        return null;
       }
 
       let data = JSON.parse(body);
       let namespaces = data.namespaces;
-      async.map(namespaces, (namespace, cb) => {
-        hubUtil.request({
-          url: `${REGHUB2_ENDPOINT}/repositories/${namespace}`
-        }, (error, response, body) => {
+
+      // Get Orgs for user
+      hubUtil.request({
+        url: `${REGHUB2_ENDPOINT}/user/orgs/?page_size=50`
+      }, (orgError, orgResponse, orgBody) => {
+        if (orgError) {
+          repositoryServerActions.error({orgError});
+          if (callback) {
+            return callback(orgError);
+          }
+          return null;
+        }
+
+        if (orgResponse.statusCode !== 200) {
+          let generalError = new Error('Failed to fetch repos');
+          repositoryServerActions.error({error: generalError});
+          if (callback) {
+            return callback({error: generalError});
+          }
+          return null;
+        }
+        let orgs = JSON.parse(orgBody);
```

Comment:
I'll update it to a try-catch; this was re-using existing code for simplification, but might as well update it.